Added image support

DustGalaxy opened the pull request:
Simple implementation of adding images. 
I might have missed something or not caught the style of the code, so I welcome criticism.

DustGalaxy committed:
adding image support

DustGalaxy committed:
Update README.md

Ae-Mc commented:
Now It's possible to add another images as binaries. But how to use them in book's body?

DustGalaxy commented:
For example, how I use the ability to add pictures:
```python
def _insert_image(self, image: Image) -> ET.Element:
    if self.with_images:
        self.book.images.append(image)
        return ET.Element("image", attrib={"xlink:href": f"#{image.uid}"})
    else:
        return ET.Element("custom")
```
I then embed that element in the right place in the xml tree.

Ae-Mc committed:
fix: generate different ids for each book

Ae-Mc committed:
feat: rework image support

Now it's possible to add it to chapters only and it will be added to binaries automatically
Simplify chapters annotation. Add ability of mixing ET.Element, Image and str in paragraphs
